collection-renderer-lib/src/cmr/collection_renderer/services/collection_renderer.clj

```diff
@@ -24,11 +26,13 @@
   "The main ERB used to generate the Collection HTML."
   (io/resource "collection_preview/collection_preview.erb"))
 
-(def preview-gem-umm-schema-version
-  "Temporay def for the UMM schema version that is supported by the CMR preview gem.
-   This will be removed once CMR preview gem is fixed to include the UMM schema version
-   it supports in the gem. (MMT-921)"
-  "1.6")
+(def preview-gem-umm-version-config-file
+  "Defines the path to the UMM schema version config file within the preview gem."
+  "gems/cmr_metadata_preview-0.0.1/.umm-version")
```

Comment:
Currently we have no supported way in MMT to provide version control. I have been in dialog with MMT folks to get that fixed. They will be making changes to start versioning their gem and this will be different once they start doing that.